gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/tasks/CreateManifestTask.java

```diff
@@ -207,6 +207,27 @@ final void setManifestFile(File manifestFile) {
     @TaskAction
     final void createManifest() throws Exception {
         validateProjectVersion();
+        jsonMapper.writeValue(
+                getManifestFile(),
+                SlsManifest.builder()
+                        .manifestVersion("1.0")
+                        .productType(productType.get())
+                        .productGroup(serviceGroup.get())
+                        .productName(serviceName.get())
+                        .productVersion(getProjectVersion())
+                        .putAllExtensions(manifestExtensions.get())
+                        .putExtensions("product-dependencies", computeProductDependencies())
+                        .putExtensions("diagnostics", findSupportedDiagnosticTypesFromClasspath())
```

Comment:
Thoughts on only including this if we've detected a diagnostic? This way it doesn't result in a change until we're rolling things out.